src/models/property.ts

```diff
@@ -0,0 +1,104 @@
+import db from "../config/db";
+import { check, validationResult } from "express-validator/check";
+import * as uuid from "uuid/v4";
+
+export type Property = {
+  id: string;
+  price: number;
+  city: string;
+  address: string;
+  description: string;
+};
+
+export const postValidator = [
+  check("price").isNumeric(),
+  check("city").isString(),
+  check("address").isString(),
+  check("description").isString()
+];
+
+export const putValidator = [
+  check("price")
+    .isNumeric()
+    .optional(),
+  check("city")
+    .isString()
+    .optional(),
+  check("address")
+    .isString()
+    .optional(),
+  check("description")
+    .isString()
+    .optional()
+];
+
+export function saveProperty(propertyPartial: Property) {
```

Comment:
I think `createProperty` would have made more sense here. Technically both creation and updating are "saving".